### PR TITLE
*/py-*: update

### PR DESCRIPTION
**Pull request by pickfire**

### CODE REVIEW

File: main/py-cryptography/APKBUILD [hunk 1]
```diff
@@ -3,7 +3,7 @@
 pkgname=py-cryptography
 _pkgname=${pkgname#py-}
 pkgver=1.7.2
```
Comment (misery) [inline main/py-cryptography/APKBUILD]: upgrade ???

by the way: https://github.com/alpinelinux/aports/pull/1001

**Comment by jirutka** [comment]
The only updated pkg in this PR is main/py-requests-oauthlib, all others have just set pkgrel to 0.

**Comment by pickfire** [comment]
@jirutka Did you set the pkgrel to 0? I thought abump automatically sets that?

**Comment by jirutka** [comment]
I've just rebased it against master, to use more up-to-date mirror. You actually didn't bump pkgver, except one package...

**Comment by pickfire** [comment]
What do I need to change now?

**Comment by jirutka** [comment]
Actually bump packages... Please look at diff here in the PR to see what you've really committed.

File: main/py-cryptography/APKBUILD (hunk 1 above)
Comment (pickfire) [inline main/py-cryptography/APKBUILD]: Ah, I didn't notice this.

**Comment by pickfire** [comment]
Ah, looks like there's a lot packages that aren't upgraded.

**Pull request closed without merging**